bleak/backends/bluezdbus/client.py

```diff
@@ -90,7 +90,7 @@ async def connect(self, **kwargs) -> bool:
         """Connect to the specified GATT server.
 
         Keyword Args:
-            timeout (float): Timeout for required ``BleakScanner.find_device_by_address`` call. Defaults to 10.0.
+            timeout (float): Defaults to 10.0.
```

Comment:
Perhaps specify the unit as seconds; so "timeout (float): Defaults to 10.0 seconds."